ovos_cli_client/__main__.py

```diff
@@ -49,10 +49,12 @@ def main():
 
     legacy_path = "/var/log/mycroft"
 
-    if 'log_dir' not in config:
-        config["log_dir"] = f"{xdg_state_home()}/{get_xdg_base()}"
```

Comment:
hum, this makes sense to me, but maybe the old key needs to be supported if it was valid under mycroft-core.... did we mess up in this repo or in ovos_utils?

maybe add a log warning if old config key exists about it being deprecated next release?

Reply:
Mycroft doesn't seem to have any way to specifiy the log path.
I couldn't find any reference either in this file or in the log.py file. So this seems to be a ovos bug only.